sdk/formrecognizer/ai-form-recognizer/src/formRecognizerClient.ts

```diff
@@ -617,8 +667,149 @@ export class FormRecognizerClient {
+   * const poller = await client.beginRecognizeInvoices(readStream, {
+   *   contentType: "application/pdf",
+   *   onProgress: (state) => { console.log(`status: ${state.status}`); }
+   * });
+   *
+   * const [invoice] = await poller.pollUntilDone();
+   * ```
+   *
+   * @summary Recognizes invoice information from a given document
+   * @param invoice Input document
+   * @param options Options for the recognition operation
+   */
+  public async beginRecognizeInvoices(
+    invoice: FormRecognizerRequestBody,
+    options: BeginRecognizeInvoicesOptions = {}
+  ): Promise<FormPollerLike> {
+    const { span } = makeSpanner("FormRecognizerClient-beginRecognizeInvoices", {
+      ...options,
+      includeTextDetails: options.includeFieldElements
+    });
+
+    const poller = new FormRecognitionPoller({
+      expectedDocType: "prebuilt:invoice",
+      createOperation: span("invoicesInternal", async (finalOptions) => {
+        const requestBody = await toRequestBody(invoice);
+        const contentType = finalOptions.contentType ?? (await getContentType(requestBody));
+        return processOperationLocation(
+          await this.client.analyzeInvoiceAsync(
+            contentType!,
+            requestBody as Blob | ArrayBuffer | ArrayBufferView,
+            operationOptionsToRequestOptionsBase(finalOptions)
+          )
+        );
+      }),
+      getResult: span("getInvoices", async (finalOptions, resultId) =>
+        this.client.getAnalyzeInvoiceResult(
+          resultId,
+          operationOptionsToRequestOptionsBase(finalOptions)
+        )
+      ),
+      ...options
+    });
+
+    await poller.poll();
+    return poller;
+  }
+
   /**
-   * Recognizes data from receipts using pre-built receipt model, enabling you to extract structure data
+   * Recognizes invoice information from a URL using a pre-built invoice model, enabling you to extract structured data
+   * from invoices such as customer address, vendor address, purchase order ID, etc.
+   *
+   * For a list of fields that are contained in the response, please refer to the documentation at the following link: https://aka.ms/azsdk/formrecognizer/invoicefields
+   *
+   * This method returns a long running operation poller that allows you to wait
+   * indefinitely until the operation is completed.
+   *
+   * Note that the onProgress callback will not be invoked if the operation completes in the first
+   * request, and attempting to cancel a completed copy will result in an error being thrown.
+   *
+   * Example usage:
+   * ```ts
+   * const url = "<url to the invoice document>";
+   * const client = new FormRecognizerClient(endpoint, new AzureKeyCredential(apiKey));
+   * const poller = await client.beginRecognizeInvoicesFromUrl(url, {
+   *   includeFieldElements: true,
+   *   onProgress: (state) => {
+   *     console.log(`analyzing status: ${state.status}`);
+   *   }
+   * });
+   * const [receipt] = await poller.pollUntilDone();
```

Comment:
I think it should be
```suggestion
   * const [invoice] = await poller.pollUntilDone();
```